### ISSUE TITLE
Teleport 6.2 post-release checklist

### ISSUE THREAD

**Issue by webvictim** [gravitational/teleport#7030]
## Post-release Checklist

This checklist is to be run after cutting a release.

### All releases

- [x] Create PR to update default Teleport version in Teleport docs
  - Example: https://github.com/gravitational/teleport/pull/4615
- [x] Create PR to update default AMI versions in Makefile and AMIs.md under https://github.com/gravitational/teleport/blob/master/assets/aws
  - Example: https://github.com/gravitational/teleport/pull/4608
- [x] Create PR to update app version and Helm chart version (https://github.com/gravitational/teleport/blob/master/examples/chart)

### Major/minor releases only

- [x] Update support matrix in docs FAQ page
- [x] Create PR to update default Docker image version in Teleport docs
  - Example: https://github.com/gravitational/teleport/pull/4615
- [x] Create PR to update default Teleport image referenced in docker/teleport-quickstart.yml and docker/teleport-ent-quickstart.yml
  - Example: https://github.com/gravitational/teleport/pull/4655
- [x] Update `CURRENT_VERSION_ROOT` and other previous versions in Drone `teleport-docker-cron` job
  - Example: https://github.com/gravitational/teleport/pull/4602

**Comment by webvictim** [comment]
>  Create PR to update default Teleport version in Teleport docs
https://github.com/gravitational/next/commit/1265dedacc70bcc5f458a407501606761beee381

**Comment by webvictim** [comment]
> * Create PR to update default AMI versions in Makefile and AMIs.md under https://github.com/gravitational/teleport/blob/master/assets/aws

#7037

**Comment by webvictim** [comment]
> [ ] Create PR to update app version and Helm chart version

No longer needed after #5582

**Comment by webvictim** [comment]
>  [ ] Update support matrix in docs FAQ page

Not needed for a minor release, Teleport 6 is already present in the FAQ.

**Comment by webvictim** [comment]
> [ ]  Create PR to update default Docker image version in Teleport docs

Not needed with the new `teleport:6` Docker image labelling.

**Comment by webvictim** [comment]
> [ ]  Create PR to update default Teleport image referenced in docker/teleport-quickstart.yml and docker/teleport-ent-quickstart.yml

#7055

**Comment by webvictim** [comment]
> [ ] Update `CURRENT_VERSION_ROOT` and other previous versions in Drone `teleport-docker-cron` job

Not needed as this is a minor release.

**Comment by webvictim** [comment]
This is all complete.
